source/tyr/tyr/tasks.py

```diff
@@ -115,41 +122,45 @@ def import_data(
         'shape': shape2ed,
     }
 
-    for _file in files:
-        filename = None
+    # For skip_2ed, skip inserting last_load_dataset files into ed database
+    if not skip_2ed:
```

Comment:
if skip_2ed:
    process_ed2nav()
    return

for _file in files:
    .....
process_ed2nav()